gcovr/tests/shared_lib/reference/coverage.html

```diff
@@ -252,12 +266,12 @@
           </tr>
           <tr>
             <td class="headerName">Date:</td>
-            <td class="headerValue">2016-08-06</td>
+            <td class="headerValue">0000-00-00</td>
             <td></td>
             <td class="headerName">Lines:</td>
             <td class="headerTableEntry">8</td>
             <td class="headerTableEntry">10</td>
-            <td class="headerTableEntry" style="background-color:LightGreen">80.0 %</td>
+            <td class="headerTableEntry" style="background-color:#FFFF55">80.0 %</td>
```

Comment:
If changing to using hex values for colors, should they _all_ be changed to hex? It doesn't affect functionality of course, but its a point to consider for consistency's sake